daml-script/runner/src/main/scala/com/digitalasset/daml/lf/engine/script/Converter.scala

```diff
@@ -262,25 +255,15 @@ object Converter {
       // typerep, contract id, choice argument and continuation
       case SRecord(_, _, vals) if vals.size == 4 =>
         for {
-          tplId <- typeRepToIdentifier(vals.get(0))
+          typeId <- typeRepToIdentifier(vals.get(0))
           cid <- toContractId(vals.get(1))
           anyChoice <- toAnyChoice(vals.get(2))
-        } yield anyChoice match {
-          case AnyChoice.Template(name, arg) =>
-            command.ApiCommand.Exercise(
-              typeId = TemplateOrInterface.Template(tplId),
-              contractId = cid,
-              choiceId = name,
-              argument = arg.toUnnormalizedValue,
-            )
-          case AnyChoice.Interface(ifaceId, name, arg) =>
-            command.ApiCommand.Exercise(
-              typeId = TemplateOrInterface.Interface(ifaceId),
-              contractId = cid,
-              choiceId = name,
-              argument = arg.toUnnormalizedValue,
-            )
-        }
+        } yield command.ApiCommand.Exercise(
+          typeId = TemplateOrInterface.Template(typeId),
```

Comment:
after this, could we change the type of `ApiCommand.Exercise.typeId` back to `Identifier`?